package.json

```diff
@@ -1751,21 +1751,20 @@
             "description": "%configuration.razor.languageServer.debug%",
             "order": 90
           },
-          "razor.trace": {
+          "razor.server.trace": {
+            "scope": "window",
             "type": "string",
-            "default": "Off",
             "enum": [
-              "Off",
-              "Messages",
-              "Verbose"
-            ],
-            "enumDescriptions": [
-              "%configuration.razor.trace.off%",
-              "%configuration.razor.trace.messages%",
-              "%configuration.razor.trace.verbose%"
+              "Trace",
```

Comment:
Should we still have enum descriptions?

Reply:
Personally, I don't think add any value.